docs/resources/streamlit.md

```diff
@@ -0,0 +1,106 @@
+---
+page_title: "snowflake_streamlit Resource - terraform-provider-snowflake"
+subcategory: ""
+description: |-
+  Resource used to manage streamlits objects. For more information, check streamlit documentation https://docs.snowflake.com/en/sql-reference/commands-streamlit.
+---
+
+!> **V1 release candidate** This resource was reworked and is a release candidate for the V1. We do not expect significant changes in it before the V1. We will welcome any feedback and adjust the resource if needed. Any errors reported will be resolved with a higher priority. We encourage checking this resource out before the V1 release. Please follow the [migration guide](https://github.com/Snowflake-Labs/terraform-provider-snowflake/blob/main/MIGRATION_GUIDE.md#v0920--v0930) to use it.
```

Comment:
migration guide should be updated to 0.93-0.94

Reply:
Fixed.